Update python doc for MultiVector data type (#1713)
infiniflow/infinity: docs/references/pysdk_api_reference.md

#### Create a table with a vector column only

:::tip NOTE
You can build a HNSW index on the vector column to speed up the match_dense search.
:::

```python
# Create a table with a vector column only:  
# - `vector`: The column is a vector column
# - `128`: The vector dimension
# - `float`: The primitive data type of the vectors. Can be `float`/`float32`, `float16`, `bfloat16`, `uint8` or `int8`
db_object.create_table("my_table", {"c1": {"type": "vector,128,float"}}, None)

```

#### Create a table with a multi-vector column only

:::tip NOTE

> You can build a HNSW index on the multi-vector column to speed up the match_dense search.

Reviewer: ```suggestion
You can build an HNSW index on the multi-vector column to accelerate match_dense search.
```
